api/v1alpha5/metal3datatemplate_types.go

```diff
@@ -202,6 +202,11 @@ type NetworkDataLinkBond struct {
 	// balance-rr, active-backup, balance-xor, broadcast, balance-tlb, balance-alb, 802.3ad
 	BondMode string `json:"bondMode"`
 
+	// +kubebuilder:validation:Enum="layer2";"layer3+4";"layer2+3"
+	// Selects the transmit hash policy used for port selection in balance-xor and 802.3ad modes
+	// +optional
+	BondHashPolicy string `json:"bond_xmit_hash_policy"`
```

Comment:
The JSON name does not conform to the used standard and does not fully match the field name. Why not just `bondHashPolicy`?